### PR TITLE
feat: messageId format updated

### PR DESCRIPTION
**Pull request by MoumitaM**
## Description of the change

> Default logic for generating message Id updated from `timestamp+ uuid` to only `uuid`.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/684)

### CODE REVIEW

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: utils/RudderMessage.js [hunk 1]
```diff
@@ -3,22 +3,26 @@ import { generateUUID } from "./utils";
...
-  // Add actual microseconds
-  // if high-precision timer if available
-  if (
-    typeof performance !== "undefined" &&
-    typeof performance.now === "function"
-  ) {
-    const pNow = performance.now();
-    const microSec = 1000 * (pNow - Math.floor(pNow));
-    ts += microSec;
+function generateMessageId(flag = false) {
```
[the remark this answers is not in the document]
Comment (MoumitaM) [reply utils/RudderMessage.js]: Just wanted to keep the logic in case of future use. Updated the name of the flag.

**Review by saikumarrs — DISMISSED** [review]

**Comment by MoumitaM** [comment]
Not required anymore.

**Comment by bardisg** [comment]
@MoumitaM is this ready to be reviewed based on latest requirements?

**Comment by MoumitaM** [comment]
> @MoumitaM is this ready to be reviewed based on latest requirements?

Yes @bardisg

**Review by bardisg — APPROVED** [review]

**Pull request merged**